src/eduid_userdb/user.py

```diff
@@ -168,19 +171,15 @@ def _parse_phone_numbers(self):
             _phones = self._data_in.pop('mobile')
             _primary = [x for x in _phones if x.get('primary', False)]
             if _phones and not _primary:
-                # None of the phone numbers are primary. Promote either the first verified
-                # entry found, or failing that - the first entry in the list.
-                _primary_set = False
+                # None of the phone numbers are primary. Promote the first verified
+                # entry found (or none if there are no verified entries).
                 for _this in _phones:
                     if _this.get('verified', False):
                         _this['primary'] = True
-                        _primary_set = True
                         break
-                if not _primary_set:
-                    _phones[0]['primary'] = True
-            self._data_in['phone'] = _phones
+            self._data_in['mobile'] = _phones
```

Comment:
Again, this looks wrong. not all phone numbers are mobiles, but all mobiles are phones, so for eduid-userdb we wanted 'phone' instead of 'mobile'.